install_ffmpeg.sh

```diff
@@ -159,6 +159,6 @@ fi
 
 if [[ ! -e "$ROOT/ffmpeg/libavcodec/libavcodec.a" || $BUILD_TAGS == *"debug-video"* ]]; then
   cd "$ROOT/ffmpeg"
-  make $FFMPEG_MAKE_EXTRA_ARGS
-  make install
+  make -j$(nproc) $FFMPEG_MAKE_EXTRA_ARGS
```

Comment:
If we adding this, `$FFMPEG_MAKE_EXTRA_ARGS` could be deleted completely, it does the same

Reply:
Hm I actually think that it is a good thing that the current usage of `$FFMPEG_MAKE_EXTRA_ARGS` allows the `-j` flag to be toggled on/off (although this is only for running make for `ffmpeg` and it is set specifically to `-j4`). IIUC the change in this PR would result in this script always maxing out processing units when running make in this file. Since this flag is run locally as a part of the development process as well, I could see scenarios where someone might not necessarily want to dedicate all available resources to compilation especially if they are doing other things at the same time. Could an alternative be to make it easy to toggle `-j$(nproc)` on/off?

Reply:
That sounds reasonable to me!